doc/guides/dts/index.rst

```diff
@@ -356,32 +359,57 @@ Example: Subset of DTS/YAML files for NXP FRDM K64F (Subject to Change)::
   dts/bindings/pinctrl/nxp,kinetis-pinmux.yaml
   dts/bindings/serial/nxp,kinetis-uart.yaml
 
-YAML bindings for device nodes
-******************************
+.. _bindings:
+
+Bindings
+********
+
+``.dts`` files describe the available hardware devices, but don't tell the
+system which pieces of information are useful, or what kind of configuration
+output (``#define``'s) should be generated. *Bindings* provide this
+information. Bindings are files in YAML format.
+
+Configuration output is only generated for devices that have bindings.
+
+Nodes are mapped to bindings via their ``compatible`` string(s). Take
+the following node as an example:
+
+.. code-block:: none
+
+   bar-device {
+   	compatible = "foo-company,bar-device";
+   	...
+   };
+
+This node would get mapped to a binding with this in it:
+
+.. code-block:: yaml
+
+   ...
+
+   properties:
+       compatible:
+           constraint: "foo-company,bar-device"
+
+       ...
 
-Device tree describes hardware and configuration, but it doesn't tell the
-system which pieces of information are useful, or how to generate configuration
-data from the device tree nodes.  For this, we rely on YAML binding files to
-describe the contents or definition of a device tree node and instruct how the
-extracted information should be formatted.
+Binding are stored in :zephyr_file:`dts/bindings/`.
```

Comment:
Maybe a word that the filename matches the compatible string?

Reply:
Added (and s/binding/bindings/)